var/spack/repos/builtin/packages/aws-parallelcluster/package.py

```diff
@@ -40,6 +41,7 @@ class AwsParallelcluster(PythonPackage):
     depends_on('py-enum34@1.1.6:', when='^python@:3.3', type=('build', 'run'))
     depends_on('py-pyyaml@5.1.2:', type=('build', 'run'))
     depends_on('py-configparser@3.5.0:3.8.1', when='^python@:2', type=('build', 'run'))
+    depends_on('py-jinja2@2.11.0:', type=('build', 'run'))
```

Comment:
It looks like some of the required versions have changed. This can be handled like so:
```python
depends_on('py-boto3@1.10.15:', when='@:2.8', type=('build', 'run'))
depends_on('py-boto3@1.14.3:', when='@2.9:', type=('build', 'run'))
```

Reply:
Hi, thank you for pointing that out, PR is updated.